tests/test_flask_util.py

```diff
@@ -303,6 +309,19 @@ def test_callback_view_errors(self):
             rv = c.get('/oauth2callback?state=%s&code=codez' % state)
             self.assertEqual(rv.status_code, httplib.BAD_REQUEST)
 
+        # Missing flow.
+        with self.app.test_client() as c:
```

Comment:
Single-letter identifiers are disallowed by our style guide. Surely there must be some more descriptive way by which we might know the value assigned here?